### PR TITLE
a better way to wait for uninstall or undeploy

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

**Comment by kensipe** [comment]
@sschneid looking for a review

File: shakedown/dcos/service.py [hunk 1]
```diff
@@ -191,6 +209,22 @@ def service_healthy(service_name, app_id=None):
...
+def mesos_task_present_predicate(task_name):
+    return get_mesos_task(task_name) is not None
+
+
+def mesos_task_not_present_predicate(task_name):
+    return get_mesos_task(task_name) is None
+
+
+def wait_for_mesos_task(task_name, timeout_sec=120):
+        return time_wait(lambda: mesos_task_present_predicate(task_name), timeout_seconds=timeout_sec)
```
Comment (sschneid) [inline shakedown/dcos/service.py]: Indenting seems off here and at line 225.

**Comment by sschneid** [comment]
👍 , this looks good aside from the minor indenting comments.

File: shakedown/dcos/service.py (hunk 1 above)
Comment (kensipe) [inline shakedown/dcos/service.py]: good catch!

**Pull request merged**